### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by youngbupark**
# Description

Improve documentation.

- Main readme.md
- Examples
- DemoActors

## Issue reference

#37 

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [x] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,13 +37,35 @@ This includes the following packages:
...
+* Official package - WIP
+
+```sh
+pip install dapr
+```
+
+* Development package
+
+```sh
+pip install dapr-dev
```
Comment (amanbha) [inline README.md:58]: will dapr-dev and dapr merge in future? Can we have a more layered approach: a dapr package which contains auto generated client apis, a dapr-actor package which contain actor apis?

Comment (youngbupark) [reply README.md:58]: most of python pkg splits two repository for daily build and official build so we will not. 

layered approach is quite complicated. unlike java and dotnet, python package zip python codes. so I am not sure that how much benefit we can get by splitting sdk to multiple ones.

Comment (youngbupark) [reply README.md:58]: Tensorflow example - 
1. nightly build - https://pypi.org/project/tf-nightly/#history
2. official build - https://pypi.org/project/tensorflow/#history

Comment (amanbha) [reply README.md:58]: makes sense, when I searach on pypi: https://pypi.org/search/?q=dapr
it seemed confusing, can the name be changed or more text can be added to description of package?

Comment (youngbupark) [reply README.md:58]: I changed the title of the packages.

File: setup.py [hunk 2]
```diff
@@ -48,8 +48,8 @@ def is_release():
 if not is_release():
     name += '-dev'
     version = f'{parse_version(__version__)}.dev{build_number}'
-    description = 'Dapr SDK for python'
-    long_description = 'This is the development build.'
+    description = 'The developmental release for Dapr Python SDK.'
```
Comment (youngbupark) [inline setup.py:51]: @amanbha I changed the description of pkg. once this pr is merged, this description will be shown in pypi pkg.

**Pull request merged**